ang/civicase/ContactsDataService.js

```diff
@@ -19,6 +19,7 @@
       'street_address',
       'tag'
     ];
+    var defer;
```

Comment:
Can we move this to the top, just before `var savedContacts = [];`. Making the list more alphabetically sorted. Also, it more readable if variable declaration w/o values are written first.